### PR TITLE
Update to the latest RSTAB

### PR DESCRIPTION
**Pull request by OndraMichal**
# Description

This includes udpdates to latest RSTAB version 6.04.0002.291.84908525896 including minor fixes.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

# How Has This Been Tested?

- [x] Unit Tests
- [ ] Attached examples

**Test Configuration**:
* RFEM / RSTAB version: 6.04.0002.291.84908525896
* Python version: 3.12.0

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: RSTAB/initModel.py [hunk 1]
```diff
@@ -289,12 +289,18 @@ def closeModel(index_or_name, save_changes = False):
...
 
         modelLs = client.service.get_model_list().name
-        Model.__delete__(Model, index_or_name)
-        client.service.close_model(modelLs.index(index_or_name), save_changes)
+        if index_or_name in modelLs:
+            try:
+                Model.__delete__(Model, index_or_name)
+                client.service.close_model(modelLs.index(index_or_name), save_changes)
+            except:
+                print('Model did NOT close closed properly.')
```
Comment (heetrojivadiya) [inline RSTAB/initModel.py:301]: the print sentence need to be corrected.

**Review by heetrojivadiya — COMMENTED** [review]
Just one comment need to be addressed else everything work fine for me.

**Review by heetrojivadiya — APPROVED** [review]

**Review by aliciaknauer — COMMENTED** [review]
Hi!
the test_plausibilitycheck is failing, but the rest works fine.

File: RSTAB/ImportExport/exports.py [hunk 2]
```diff
@@ -30,7 +30,7 @@ def ExportResultTablesToXML(targetFilePath: str, model = Model):
     '''
     model.clientModel.service.export_result_tables_to_xml(targetFilePath)
 
-def ExportResultTablesWithDetailedMembersResultsToCSV(targetDirectoryPath: str, model = Model):
+def ExportResultTablesWithDetaliedMembersResultsToCSV(targetDirectoryPath: str, model = Model):
```
Comment (heetrojivadiya) [inline RSTAB/ImportExport/exports.py:33]: Typo in detailed

File: RSTAB/LoadCasesAndCombinations/resultCombination.py [hunk 3]
```diff
@@ -25,6 +41,47 @@ def __init__(self,
...
+            clientObject.items = model.clientModel.factory.create('ns0:result_combination.items')
+            for i,j in enumerate(combination_items):
+                rci = model.clientModel.factory.create('ns0:result_combination_items_row')
+                rci.no = i+1
+                rci.row = model.clientModel.factory.create('ns0:result_combination_items')
+                clearAttributes(rci.row)
+                rci.row.case_object_item = combination_items[i][0]
+                rci.row.operator_type = combination_items[i][1].name
+                rci.row.case_object_factor = combination_items[i][2]
+                rci.row.case_object_load_type = combination_items[i][3].name
```
Comment (heetrojivadiya) [inline RSTAB/LoadCasesAndCombinations/resultCombination.py:66]: If parenthesis also available in RSTAB like RFEM then you may update in RSTAB too like in RFEM. 
https://github.com/Dlubal-Software/RFEM_Python_Client/pull/292/files

File: RSTAB/ImportExport/exports.py (hunk 2 above)
Comment (OndraMichal) [reply RSTAB/ImportExport/exports.py:33]: fixed

File: RSTAB/ImportExport/exports.py [hunk 4]
```diff
@@ -39,7 +39,7 @@ def ExportResultTablesWithDetailedMembersResultsToCSV(targetDirectoryPath: str,
     '''
     model.clientModel.service.export_result_tables_with_detailed_members_results_to_csv(targetDirectoryPath)
 
-def ExportResultTablesWithDetailedMembersResultsToXML(targetFilePath: str, model = Model):
+def ExportResultTablesWithDetaliedMembersResultsToXML(targetFilePath: str, model = Model):
```
[the remark this answers is not in the document]
Comment (OndraMichal) [reply RSTAB/ImportExport/exports.py:42]: fixed

File: RSTAB/LoadCasesAndCombinations/resultCombination.py (hunk 3 above)
Comment (OndraMichal) [reply RSTAB/LoadCasesAndCombinations/resultCombination.py:66]: Addressed

**Review by aliciaknauer — APPROVED** [review]

**Pull request merged**